ofrak_core/ofrak/gui/server.py

```diff
@@ -485,6 +575,11 @@ async def get_all_tags(self, request: Request) -> Response:
             self._serializer.to_pjson(self._ofrak_context.get_all_tags(), Set[ResourceTag])
         )
 
+    @exceptions_to_http(SerializedError)
+    async def get_script(self, request: Request) -> Response:
+        resource = await self._get_resource_for_request(request)
+        return json_response(await self.script_builder.get_script(resource))
```

Comment:
I have not debugged why, but this is currently returning one single script for all root resources. It's supposed to be returning a different script for each root resource.

In other words, this is currently not working.

Reply:
This should be resolved now. The issue was that the same instances of ScriptActions and ScriptSessions were being shared by multiple files.